src/SensioLabs/Melody/Handler/AuthenticableHandlerInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace SensioLabs\Melody\Handler;
+
+use SensioLabs\Melody\Configuration\UserConfiguration;
+use SensioLabs\Melody\Exception\InvalidCredentialsException;
+
+/**
+ * Interface AuthenticableHandlerInterface.
+ *
+ * @author Maxime STEINHAUSSER <maxime.steinhausser@gmail.com>
+ */
+interface AuthenticableHandlerInterface extends ResourceHandlerInterface
```

Comment:
the `AuthenticableHandlerInterface` should not extends `ResourceHandlerInterface`.
And so GithubHandler should implements both interface explicitly.